src/briefcase/platforms/linux/system.py

```diff
@@ -1139,18 +1163,43 @@ def _package_rpm(self, app: AppConfig, **kwargs):  # pragma: no-cover-if-is-wind
                     else:
                         f.write(f'"/{path}"\n')
 
+                changelog_name_formats = [
+                    "CHANGELOG",
+                    "HISTORY",
+                    "NEWS",
+                    "RELEASES",
+                    "CHANGELOG.md",
+                    "CHANGELOG.rst",
+                    "CHANGELOG.txt",
+                    "HISTORY.md",
+                    "HISTORY.rst",
+                    "HISTORY.txt",
+                    "NEWS.md",
+                    "NEWS.rst",
+                    "NEWS.txt",
+                    "RELEASES.md",
+                    "RELEASES.rst",
+                ]
+                matched_changelog_name = None
+                changelog_source = None
+
                 # Add the changelog content to the bottom of the spec file.
                 f.write("\n%changelog\n")
-                changelog_source = self.base_path / "CHANGELOG"
-                if not changelog_source.is_file():
+                for name in changelog_name_formats:
+                    changelog_source = self.base_path / name
+                    if changelog_source.is_file():
+                        matched_changelog_name = name
+                        break
+
+                if matched_changelog_name is None:
                     raise BriefcaseCommandError(
                         """\
-Your project does not contain a CHANGELOG file.
-
-Create a file named `CHANGELOG` in the same directory as your `pyproject.toml`
-with details about the release.
-"""
+        Your project does not contain a CHANGELOG file.
```

Comment:
The indentation here means the error message will start with 8 spaces when printed on the console.

Also - it references a single filename (CHANGELOG), but the code now accepts multiple valid names. The error message should reflect the fact that there are multiple valid inputs.